util/db/db.go

```diff
@@ -37,6 +37,13 @@ type ArgoDB interface {
 
 	// ListHelmRepoURLs lists configured helm repositories
 	ListHelmRepos(ctx context.Context) ([]*appv1.HelmRepository, error)
+
+	// ListRepoCerticifates lists all configured certificates
+	ListRepoCertificates(ctx context.Context, selector *CertificateListSelector) (*appv1.RepositoryCertificateList, error)
```

Comment:
minor - but I think we should use `cert` abbreviation throughout the code base

Reply:
You mean like, rename all symbols from `Certificate` to the shorter `Cert`? So in the example of commented section of code, `ListRepoCertificates` would become `ListRepoCerts`, `CertificateListSelector` would become `CertListSelector` and `RepositoryCertificateList` would become `RepositoryCertList` (or even `RepoCertList`) etc etc?

Reply:
correct - I think most people would understand what a cert is, and it'll make the code simpler - this is a minor mind you

Reply:
I think this would make the code shorter, but not necessarily simpler :) However, since this would mean quite a lot of refactoring work and I think my tools won't do that automatically, I'd prefer doing this in a separate PR.

Reply:
sure, I don't feel strongly about it